client/src/components/Link/Link.test.js

```diff
@@ -1,40 +1,38 @@
 import React from 'react';
```

Comment:
I think we are safe to simply ignore the href requirement for the sake of this test.

While not ideal, we treat this link a bit differently to plain links.

```
/* eslint-disable jsx-a11y/anchor-is-valid */
```